src/backend/routers/chat.py

```diff
@@ -97,6 +97,7 @@ async def chat(
     chat_request: CohereChatRequest,
     request: Request,
     agent_id: str | None = None,
+    deployment_name: str | None = Header(default=None),
```

Comment:
Can we add a # TODO comment to have request validator use this param?

Reply:
@tianjing-li I have added a comment as requested.